style.css

```diff
@@ -0,0 +1,264 @@
+/* css reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  line-height: 1.5;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+#header {
+  display: flex;
+}
+
+#nav-bar {
+  position: fixed;
+  top: 0;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  overflow: hidden;
+}
+
+/*
+    @media only screen and (max-width:600px){
+    #nav-bar ul {
+      display: flex;
+    }
+    #nav-bar ul li {
+      padding: 1rem 2rem;
+    }
+    #nav-bar ul li a {
+      color: #333;
+      font-size: 1.2rem;
+    }     } */
+
+/* UTILITY CLASSES */
+#container {
+  max-width: 960px;
+  margin: 2rem auto 0;
+  overflow: hidden;
+}
+
+.logo {
+  width: 300px;
+  cursor: pointer;
+}
+
+@media (min-width: 1200px) {
+  body {
+    background-color: rgb(255, 255, 255);
+  }
+}
+
+.btn {
+  display: inline-block;
+  padding: 0.4rem 1.5rem;
+  color: #333;
+  text-transform: uppercase;
+  font-weight: bold;
+  background-color: #f1c40f;
+}
+
+.nav-link {
+  float: left;
+  display: block;
+  color: #000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.message-box {
+  margin: 1.5rem;
+  padding: 0.8rem;
+  text-align: center;
+}
+
+.message-box h1 {
+  font-size: 1.7rem;
+  margin-bottom: 1rem;
+}
+
+.message-box input {
+  display: block;
+  margin: auto;
+  width: 30%;
+  padding: 0.4rem;
+  margin-bottom: 1rem;
+}
+
+#features {
+  padding: 1rem;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+  display: flex;
+  flex-direction: column;
+}
+
+#features .feature {
+  display: flex;
+  justify-self: center;
+  align-items: center;
+}
+
+#features .feature i {
+  color: #ff8c00;
+}
+
+#features .feature div {
+  padding: 1rem;
+  margin-left: 3rem;
+}
+
+#features .feature div h2 {
+  font-size: 1.5rem;
+  margin-bottom: 0.5rem;
+}
+
+#how-it-work {
+  padding: 3rem;
+  margin: auto;
+  text-align: center;
+}
+
+#how-it-work #video {
+  width: 600px;
+  height: 340px;
+}
+
+#pricing {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 3rem 0.2rem;
+}
+
+.price-box {
+  border: solid #333 3px;
+  border-radius: 2px;
+  text-align: center;
+}
+
+.price-box:first-child {
+  margin-right: 1.2rem;
+}
+
+.price-box:last-child {
+  margin-left: 1.2rem;
+}
+
+.price-box h3 {
+  background-color: #ddd;
+  font-size: 1.05rem;
+  padding: 1rem;
+}
+
+.price-box h2 {
+  padding: 1rem;
+}
```

Comment:
Don’t you think that ".price-box h2" to come before selector "#features .feature div h2"  is great.

Reply:
- [ ] Good job pointing out the order priority 👍. However, it is good to include an explanation as to why a change is required with reasons why you think they should make the changes politely, as it will help the developer to see the importance of the change, and help them be a better developer. Please, let us edit this comment to be polite and to include why re-arranging the selectors is a good practice.

Reply:
Good job, I think that ".price-box h2" to come before selector "#features .feature div h2" is great. Stylesheets are most legible when overriding selectors always come after the selectors they override. If a selector with higher specificity comes before the selector it overrides, we have to think harder to understand it, because it violates the source order expectation. It’s likely going  to give an error message aspect you turn off the no-descending-specificity rule.